components/Form.js

```diff
@@ -14,78 +25,101 @@ export default function CreateForm({ onCreateItem }) {
-              );
-            })}
-          </StyledCategorySelect>
-        </StyledCategorySelectContainer>
-
-        <StyledQuantityInputContainer>
-          <label htmlFor="quantity">Quantity</label>
-          <StyledQuantityInput
-            name="quantity"
-            id="quantity"
-            type="number"
-            placeholder="(Required)"
-            min="0"
-            max="999"
-            required
-          />
-        </StyledQuantityInputContainer>
-      </FlexContainer>
-      <label htmlFor="comment">Comment</label>
-      <StyledCommentTextarea
-        name="comment"
-        id="comment"
-        placeholder="(Optional)"
-        maxLength="100"
-        wrap="hard"
-      ></StyledCommentTextarea>
-      <StyledCreateButton>Create</StyledCreateButton>
-    </StyledForm>
+    <FormContainer>
+      <StyledForm onSubmit={handleSubmit}>
+        {!editItem && <StyledHeader>Add new item</StyledHeader>}
+        {editItem && <StyledHeader>Edit shopping item</StyledHeader>}
+        <label htmlFor="name">Item Name</label>
+        <StyledItemNameInput
+          defaultValue={editItem ? editItem.name : ""}
+          name="name"
+          id="name"
+          type="text"
+          placeholder="(Required)"
+          maxLength="20"
+          required
+        />
+        <FlexContainer>
+          <StyledCategorySelectContainer>
+            <label htmlFor="category">Category</label>
+            <StyledCategorySelect
+              name="category"
+              id="category"
+              value={selectedCategory}
+              onChange={handleCategoryChange}
+              required
+            >
+              <option value="">---Choose a category---</option>
+              {categories.map((category) => {
+                return (
+                  <option key={category} value={category}>
+                    {category}
+                  </option>
+                );
+              })}
+            </StyledCategorySelect>
+          </StyledCategorySelectContainer>
+
+          <StyledQuantityInputContainer>
+            <label htmlFor="quantity">Quantity</label>
+            <StyledQuantityInput
+              defaultValue={editItem ? editItem.quantity : ""}
+              name="quantity"
+              id="quantity"
+              type="number"
+              placeholder="(Required)"
+              min="0"
+              max="999"
+              required
+            />
+          </StyledQuantityInputContainer>
+        </FlexContainer>
+        <label htmlFor="comment">Comment</label>
+        <StyledCommentTextarea
+          defaultValue={editItem ? editItem.comment : ""}
+          name="comment"
+          id="comment"
+          placeholder="(Optional)"
+          maxLength="100"
+          wrap="hard"
+        ></StyledCommentTextarea>
+        {!editItem && <StyledCreateButton>Create</StyledCreateButton>}
+        {editItem && <StyledSubmitButton>Submit</StyledSubmitButton>}
```

Comment:
This can also be simplified using a ternary operator
```suggestion
     {editItem ? (
  <StyledSubmitButton>Submit</StyledSubmitButton>
) : (
  <StyledCreateButton>Create</StyledCreateButton>
)}
```